src/server/common/helpers/decorators/repository.js

```diff
@@ -1,10 +1,10 @@
 import { omit } from 'lodash'
 
-function repositoryDecorator(service, repository) {
+function repositoryDecorator(service, repository = {}) {
   return omit(
     {
       ...service,
-      ...(repository && repository)
+      ...repository
```

Comment:
Is this the way? I thought sonar was just wrong.

Reply:
Note I have added a default of an empty object above, so we can just spread the object as it will always be an object